### ISSUE TITLE
etcher is signed with invalid signature

### ISSUE THREAD

**Issue by searedvandal** [Antergos/antergos-packages#347]
Etcher seems to be signed with a key that's not valid.

I tried updating the antergos-keyring package, and all other tricks I could find on the arch wiki. And still not working. This is the only package that seems to have this issue. Others have been reporting the same thing on irc.

```
$ sudo pacman -S etcher
resolving dependencies...
looking for conflicting packages...

Package (1)      New Version  Net Change  Download Size

antergos/etcher  1.4.4-1      209.61 MiB      41.13 MiB

Total Download Size:    41.13 MiB
Total Installed Size:  209.61 MiB

:: Proceed with installation? [Y/n] 
:: Retrieving packages...
 etcher-1.4.4-1-x86_64                               41.1 MiB  1473K/s 00:29 [###########################################] 100%
(1/1) checking keys in keyring                                               [###########################################] 100%
(1/1) checking package integrity                                             [###########################################] 100%
error: etcher: signature from "Antergos Build Server (Automated Package Build System) <admin@antergos.org>" is invalid
:: File /var/cache/pacman/pkg/etcher-1.4.4-1-x86_64.pkg.tar.xz is corrupted (invalid or corrupted package (PGP signature)).
Do you want to delete it? [Y/n] 
error: failed to commit transaction (invalid or corrupted package (PGP signature))
Errors occurred, no packages were upgraded.
```

**Comment by killajoe** [comment]
@karasu needs to be rebuilded also?

**Comment by karasu** [comment]
https://github.com/Antergos/antergos-packages/commit/8d2cd05e4865318af8a9923762b39d971a5246e6

**Comment by tr0yspradling** [comment]
Why was this issue closed? Still affecting v1.4.4-1.

**Comment by tr0yspradling** [comment]
Why was this issue closed? Still affecting 1.4.4-1.
